min_depth : 0.3
max_depth : 3.0
N_planes : 50
confidence_map_cushion :  4
gauss_filter_size : 5
gauss_filter_sigma : 2
median_filter_radius : 7
